src/api.js

```diff
@@ -547,6 +572,15 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
         return true;
     };
 
+    Statement.prototype.bindBigInt = function bindBigInt(num, pos) {
+        if (pos == null) {
+            pos = this.pos;
+            this.pos += 1;
+        }
+        this.db.handleError(sqlite3_bind_int64(this.stmt, pos, num));
```

Comment:
Does this work ? Don't you need to convert the integer to a string for it to pass the wasm frontier ?

Also, isn't this dead code ? The method name isn't quoted so it's not usable from the outside.

Reply:
I think the best option is to remove it since BigInt is not supported at WASM level and yes I think this is a dead code since on bindValue for bigint, bindString is called.